src/common/components/select-repository-search/index.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component, PropTypes } from 'react';
+import styles from './search.css';
+
+export default class SelectRepositorySearch extends Component {
```

Comment:
Also I would consider this as a more reusable 'search' component, as we could use it to search anything else, so I'd rename it to 'SearchInput' or something similar (not related to repos).